### PR TITLE
FAQ: Change from 10 minute contact duration to 5 minutes

### PR DESCRIPTION
**Pull request by dsarkar**
Closes https://github.com/corona-warn-app/cwa-website/issues/914

### CODE REVIEW

**Comment by dsarkar** [comment]
- image

-  image 

-  image

**Comment by dsarkar** [comment]
image

---

image

---

image

File: src/data/faq.json [hunk 1]
```diff
@@ -723,7 +723,7 @@
...
                             "Each of these windows must meet certain criteria to be considered:",
-                            "<ul><li>The signal attenuation must have been below 73dB for at least 10 minutes within the 30-minute window</li>",
+                            "<ul><li>The signal attenuation must have been below 73dB for at least 10 minutes (reduced to 5 minutes on Feb 23, 2021, see <a href='../blog/2021-02-23-corona-warn-app-risk-calculation-optimization/' target='_blank' rel='noopener noreferrer'>blog post</a>) within the 30-minute window</li>",
                             "<li>The transmission risk level must be at least III (3). It is determined when sharing the diagnosis codes per day according to defined criteria such as symptom status and/or symptom onset</li></ul>",
                             "If these criteria are <b>not</b> met, an encounter is <b>not count</b> within the app. It is a 'non-risk encounter'.  However, in the operating system's contact log, a count is performed. However, this is not an error.",
```
Comment (MikeMcC399) [inline src/data/faq.json:728]: <b>not count</b>
should be
<b>not counted</b>

File: src/data/faq.json [hunk 2]
```diff
@@ -113,7 +113,7 @@
...
                             "The remaining encounters are rated collectively according to their distance and duration as well as the presumed infectiousness of the other person(s). This rating doesn't always yield an increased risk. In this case those encounters will be shown in the Corona-Warn-App but the risk status will stay the same.",
                             "For more details about risk assessment, including a sample calculation, see <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md' target='_blank' rel='noopener noreferrer'>https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md</a>."
```
Comment (MikeMcC399) [inline src/data/faq.json:118]: The documentation cwa-risk-assessment.md is outdated.
The other two FAQ articles refer to https://github.com/corona-warn-app/cwa-documentation/blob/master/images/risk_calculation/risk_calculation_enf_v2_overview.pdf for instance.

**Review by MikeMcC399 — APPROVED** [review]
The text changes themselves look fine. I have made some other comments about out of date references and one typo.

Let me know if you want this put into one or more separate issues.

File: src/data/faq.json (hunk 2 above)
Comment (mtb77) [reply src/data/faq.json:118]: Needs to be updated in a second PR

**Review by mtb77 — COMMENTED** [review]
Please see the changes requested in the PR

**Comment by GisoSchroederSAP** [comment]
I just learned today **not** to mix the 5min time-frame with the 15min threshold. The 5min relates to the total technical active scanning time realized by the ENF, but the 15min threshold is the (currently) defined "relevant contact time" that is also used by the health authorities to decide whether or not a contact is qualified as "contacted" - but we have this and other documents currently on review already. Latest target date is mid of March.

**Review by MikeMcC399 — COMMENTED** [review]
Also in line 734 of faq_de.json
"Basierend auf dem Transmission Risk Level (III bis VIII) wird nur ein Faktor zwischen 0,6 und 1,6 ermittelt"

"nur" should be "nun".

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json]: thanks

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json:118]: separate issue https://github.com/corona-warn-app/cwa-website/issues/917

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review. I created a separate issue #917.

**Comment by MikeMcC399** [comment]
@dsarkar 
I also created a separate issue #918 regarding the reference materials.

Many thanks for correcting the typos which were not part of your changes! :+1

**Pull request merged**